src/bloqade/pyqrack/squin/runtime.py

```diff
@@ -41,6 +41,21 @@ def broadcast_apply(self, qubits: ilist.IList[PyQrackQubit, Any], **kwargs) -> N
             self.apply(*targets, **kwargs)
 
 
+@dataclass(frozen=True)
+class ResetRuntime(OperatorRuntimeABC):
+    """Reset the qubit to |0> state"""
+
+    @property
+    def n_sites(self) -> int:
+        return 1
+
+    def apply(self, *qubits: PyQrackQubit, adjoint: bool = False) -> None:
+        for qubit in qubits:
+            if not qubit.is_active():
+                continue
+            qubit.sim_reg.force_m(qubit.addr, 0)
```

Comment:
Oh we have had issues with `force_m` so don't use it you can simply measure the qubit and if the measurement is `1` then just flip the state using an `x` operation on that qubit.